src/test_lib_json/main.cpp

```diff
@@ -1,9 +1,10 @@
-// Copyright 2007-2010 Baptiste Lepilleur
```

Comment:
Sorry, but I wouldn't feel right approving a commit that removes Baptiste's name from any file.